src/Parser/HolyPriest/TALENT_DESCRIPTIONS.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+
+export default {
+  descriptions: {
```

Comment:
These should describe the talents like you would in a Wowhead guide. For the Holy Paladin descriptions I copied these descriptions straight from it. This gives people more info than what they can see in the tooltip already.